docs/topics/OAuth2.md

```diff
@@ -433,6 +431,24 @@ Returns the bot's OAuth2 [application object](#DOCS_TOPICS_OAUTH2/application-ob
 }
 ```
 
+###### Application Flags
+
+| Value   | Name                             |
+|---------|----------------------------------|
+| 1 << 2  | MANAGED_EMOJI                    |
```

Comment:
Might make sense to include field descriptions here to keep this consistent with how enum flags are usually documented throughout the docs

Reply:
every application seems to have `RPC_HAS_CONNECTED` 🤔 so I'm not sure what that does

I'm not really sure about `MANAGED_EMOJI` and `GROUP_DM_CREATE` either, I expected gamewisp to have `MANAGED_EMOJI` but it doesn't

ik the others tho

Reply:
I imagine group DM create is whitelisting for creating more then 10 active group DMs.